src/components/Calculator.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+import Header from './Header';
+import Display from './Display';
+import ButtonContainer from './ButtonContainer';
+import calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const [calcObject, setCalcObject] = useState({
+    num1: null,
+    num2: null,
+    displayValue: null,
+    operator: null,
+  });
+
+  const handleButtonClick = (input) => {
+    const newCalcObject = calculate(input, calcObject);
+    setCalcObject(newCalcObject);
+  };
+
+  return (
+    <>
+      <div className="">
+        <div className="">
+          <div className="">
```

Comment:
```suggestion
      <div>
        <div>
          <div>
```


Please do this for all your code.